sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Conflict.java

```diff
@@ -42,7 +42,7 @@ public String getOperationKind() {
      *
      * @return the resource type.
      */
-    public String getResouceType() {
+    public String getResourceType() {
```

Comment:
Is it supposed to be non-public?

Reply:
Yes, implementation detail. Its fine if it is public . .